rdt-open-reader/src/main/java/io/ona/rdt/application/RDTApplication.java

```diff
@@ -68,6 +72,11 @@ public void onCreate() {
         if (sharedPreferences.getPreference(IS_IMG_SYNC_ENABLED).isEmpty()) {
             sharedPreferences.savePreference(IS_IMG_SYNC_ENABLED, String.valueOf(true));
         }
+
+        // set locale initially
+        if (StringUtils.isEmpty(sharedPreferences.getPreference(AllConstants.LANGUAGE_PREFERENCE_KEY))) {
+            sharedPreferences.savePreference(Constants.Locale.LOCALE, BuildConfig.LOCALE);
```

Comment:
The key here also needs to change to `AllConstants.LANGUAGE_PREFERENCE_KEY`.

Reply:
Done